### DISCUSSION TITLE
Setting Defaults for all Fancyboxs

### DISCUSSION THREAD

**Discussion by Ristophonics** [fancyapps/ui#105] [Q&A] [answered]
I am trying to set the default Toolbar for all fancyboxs with the following code at the end of the body tag but nothing results.. help please.

`<script>
Fancybox.bind("[data-fancybox]", {
  Toolbar: {
    display: [
      { id: "prev", position: "center" },
      { id: "counter", position: "center" },
      { id: "next", position: "center" },
      "zoom",
      "slideshow",
      "fullscreen",
      "download",
      "thumbs",
      "close",
    ],
  }
})
console.log("Setting Fancy App Defaults")
</script>`

**Comment by fancyapps** [comment]
Hi,

You are missing `;` before `console.log()` - https://fancyapps.com/playground/Jr

I recommend using decent code editor, for example, https://code.visualstudio.com/ - that shows code syntax errors and helps you with developement.

**Reply by Ristophonics** [reply]
Hmm, I am using VScode and it doesnt show the syntax error... 

![wtfnoerror](https://user-images.githubusercontent.com/36245955/131709036-4cbec2e7-ddab-40ba-abd8-70f4baf38d4f.jpg)
(not sure why VScode isn't picking those up)

Okay I added the ; and a missing comma on line 171...

`<script type="text/javascript">
Fancybox.bind("[data-fancybox]", {
  Toolbar: {
    display: [
      { id: "prev", position: "center" },
      { id: "counter", position: "center" },
      { id: "next", position: "center" },
      "zoom",
      "slideshow",
      "fullscreen",
      "download",
      "thumbs",
      "close",
    ],
  },
});
console.log("Setting FancyApp Defaults");
</script>`

 It appears to not be making changes still.  Thanks for the responses!

**Comment by fancyapps** [comment, marked as answer]
Well, maybe it is VS Code ESLint extension that shows warning.

Anyway, I have no idea why it is not working for you. Could you, please, provide a link to your page or some demo? I really would like to see and check.

**Reply by Ristophonics** [reply]
Yes sir, https://aecomviz.com/fancyhelp/

**Reply by fancyapps** [reply]
Oh, I see ... Well, I thought the toolbar is only useful for images, so it is disabled automatically if no images are detected in the gallery. Add `autoEnable: false` to disable this check, and the toolbar will be displayed for any type of content.

**Reply by Ristophonics** [reply]
lovely! Thank you.
